sacred/config/config_files.py

```diff
@@ -35,7 +35,12 @@ def __init__(self, load, dump, mode):
 
 def get_handler(filename):
     _, extension = os.path.splitext(filename)
-    return HANDLER_BY_EXT[extension]
+    try:
+        return HANDLER_BY_EXT[extension]
+    except KeyError:
+        raise ValueError(
+            'Configuration file "{}" has invalid or unsupported extension "{}".'.format(filename, extension)
```

Comment:
```suggestion
        raise ValueError(
            'Configuration file "{}" has invalid or unsupported extension "{}".'.format(
                filename, extension
            )
        )
```

Reply:
apparently that line is to long for flake8

Reply:
OK, I reduced the line length to be less than 79.